File: Kody-master/klasa 2/cpp/liczby.cpp
/*
 * liczby.cpp
 */


#include <iostream>
#include <iomanip>

using namespace std;

int liczby2()
{
   int ile = 0;
   
   for (int i = 1; i < 10; i++)
    {
        for(int j = 0; j < 10; j++)
        {
            if (i != j)
            {
            cout << i << j << " ";
            ile++;
            }
        }
        cout << endl;
    }
    return ile;
}

int liczby3()
{
   int ile = 0;
   
   for (int i = 1; i < 10; i++)
    {
        for(int j = 0; j < 10; j++)
        {
            for (int k = 0; k < 10; k++)
            {
                if (i != j && i != k && j != k)
                {
                    cout << i << j << k << " ";
                    ile++;
                }
            }
        }
        cout << endl;
    }
    return ile;
}

int main(int argc, char **argv)
{
    cout << "\n\nLiczb 2-cyfrowych: " << liczby2() << endl;
    cout << "\n\nLiczb 3-cyfrowych: " << liczby3() << endl;
    
	return 0;
}

File: klasa 2/python/liczby .py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby.py

def liczby2():
    
    """
    Drukuje wszystkie liczby 2-cyfrowe o niepowtarzających się
    cyfrach i ich ilość
    """
    ile=0
    
    for i in range(1, 10):
        for j in range(0, 10):
            if i != j:
                print("{}{} ".format(i, j), end='')
            ile += 1
    return ile
       
def liczby3():
    
    """
    Drukuje wszystkie liczby 3-cyfrowe o niepowtarzających się
    cyfrach i ich ilość
    """
    
    ile=0
    
    for i in range(1, 10):
        for j in range(0, 10):
            for k in range(0, 10):
                if i != j and i != k and j != k:
                    print("{}{}{} ".format(i, j, k), end='')
                ile += 1
    return ile
        
def main(args):
    return 0
    
if __name__ == '__main__':
    import sys
sys.exit(main(sys.argv))
